support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -1080,8 +1085,6 @@ function CheckoutComponent({
 			</Box>
 			<form
 				ref={formRef}
-				action="/contribute/recurring/create"
```

Comment:
This action is not used because we `event.preventDefault` in the onSubmit. So having this (which is actually a different URL to what we're posting to in the `formOnSubmit` function) felt confusing

Reply:
Good call 👍